packages/components/src/PlaceCard/Placeholder.js

```diff
@@ -37,28 +38,28 @@ const Placeholder = () => (
 
 const styles = StyleSheet.create({
   container: {
-    height: 152,
+    height: tokens.heightPlaceCard,
```

Comment:
I don't think so. Because it has the same height as `PlaceCard` and `Placeholder` is sub-component of `PlaceCard` (at least according to directory naming)

Reply:
Ok, I just thought so because of that unused `heightPlaceholder` value in tokens. Sorry